src/cdk-experimental/popover-edit/table-directives.ts

```diff
@@ -99,6 +104,7 @@ export class CdkEditable implements AfterViewInit, OnDestroy {
   host: {
     'tabIndex': '0',
     '[attr.aria-haspopup]': 'true',
+    '[class.cdk-popover-edit-cell]': 'true',
```

Comment:
You can just do 
```ts
  'class': 'cdk-popover-edit-cell',
```

If `class` isn't a binding, the classes will be unioned and not overwritten